UKHomeOffice/sgmr-service
# SPCR-536 Clear people formData on save and exit

### AC / Description of changes
When users go to ‘Add new person’ screen straight following editing a person, the new person screen has fields pre-filled with the previously edited person’s details.

To recreate the problem:
Update a person
Click ‘Exit without saving’
Click ‘Add new person’

### To Test
- Go to /people
- Update a person
- Click ‘Exit without saving’
- Click ‘Add new person’
- > The form should be blank

### Notes

---

**Remember to merge to feature branches not `master`.**
